src/sync/pull-request.ts

```diff
@@ -41,41 +40,31 @@ type PullRequestWithCursor = { cursor: number } & Octokit.PullsListResponseItem;
 
 export const getPullRequestTask = async (
 	logger: LoggerWithTarget,
-	github: GitHubAPI,
+	_oldGithub: GitHubAPI,
 	newGithub: GitHubInstallationClient,
-	jiraHost: string,
+	_jiraHost: string,
```

Comment:
Had to rename these vars, as they aren't used anymore (couldn't remove them from the signature completely, because they are still used in other implementations)